tests/wake-format/basic/basic.wake

```diff
@@ -623,6 +623,8 @@ def x + y = add x y
 def tool (dir: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) (addHeaders: String) =
```

Comment:
The description has a double-space or two between the arguments, but the test line doesn't; I don't know if that's being checked anywhere else, but it seems like something which might be nice to include if not.

Reply:
was a typo but I can add it as a test case